app/addListing/service.js

```diff
@@ -0,0 +1,81 @@
+function service($http, endPoint, UserService) {
+    return {
+        addTreatmentCenterSignUp: function ($scope, auth, formData) {
```

Comment:
Do not pass `$scope` around. Give the method what it needs, give no more than that.